code/AssetLib/glTF2/glTF2Exporter.cpp

```diff
@@ -870,6 +870,7 @@ void glTF2Exporter::ExportMaterials() {
             }
         }
 
+        GetMatTex(mat, m->normalTexture, aiTextureType_DIFFUSE_ROUGHNESS);
```

Comment:
Are you sure you want to assign diffuse roughness to the normal texture? This doesn't make sense to me...

Reply:
Sorry，it was a mistake and has been fixed.